doc/release/1.8.0-notes.rst

```diff
@@ -50,80 +86,808 @@ New features
+results for large positive and negative inputs, so it avoids the problems
+that would occur in the naive implementation ``log(expit(x))``.
+
+A suite of five new functions for elliptic integrals: 
+``scipy.special.ellipr{c,d,f,g,j}``. These are the
+`Carlson symmetric elliptic integrals <https://dlmf.nist.gov/19.16>`_, which
+have computational advantages over the classical Legendre integrals. Previous
+versions included some elliptic integrals from the Cephes library
+(``scipy.special.ellip{k,km1,kinc,e,einc}``) but was missing the integral of
+third kind (Legendre's Pi), which can be evaluated using the new Carlson
+functions. The new Carlson elliptic integral functions can be evaluated in the
+complex plane, whereas the Cephes library's functions are only defined for
+real inputs.
+
+Several defects in `scipy.special.hyp2f1` have been corrected. Approximately
+correct values are now returned for ``z`` near ``exp(+-i*pi/3)``, fixing
+`#8054 <https://github.com/scipy/scipy/issues/8054>`_. Evaluation for such ``z``
+is now calculated through a series derived by
+`López and Temme (2013) <https://arxiv.org/abs/1306.2046>`_ that converges in
+these regions. In addition, degenerate cases with one or more of ``a``, ``b``,
+and/or ``c`` a non-positive integer are now handled in a manner consistent with
+`mpmath's hyp2f1 implementation <https://mpmath.org/doc/current/functions/hypergeometric.html>`_,
+which fixes `#7340 <https://github.com/scipy/scipy/issues/7340>`_. These fixes
+were made as part of an effort to rewrite the Fortran 77 implementation of
+hyp2f1 in Cython piece by piece. This rewriting is now roughly 50% complete.
 
 `scipy.stats` improvements
 ==========================
 
-Hypothesis Tests
-----------------
+`scipy.stats.qmc.LatinHypercube` introduces two new optional keyword-only
+arguments, ``optimization`` and ``strength``. ``optimization`` is either
+``None`` or ``random-cd``. In the latter, random permutations are performed to
+improve the centered discrepancy. ``strength`` is either 1 or 2. 1 corresponds
+to the classical LHS while 2 has better sub-projection properties. This
+construction is referred to as an orthogonal array based LHS of strength 2.
+In both cases, the output is still a LHS.
+
+`scipy.stats.qmc.Halton` is faster as the underlying Van der Corput sequence
+was ported to Cython.
 
+The ``alternative`` parameter was added to the ``kendalltau`` and ``somersd``
+functions to allow one-sided hypothesis testing. Similarly, the masked
+versions of ``skewtest``, ``kurtosistest``, ``ttest_1samp``, ``ttest_ind``,
+and ``ttest_rel`` now also have an ``alternative`` parameter.
 
-Sample statistics
------------------
+Add `scipy.stats.gzscore` to calculate the geometrical z score.
 
+Random variate generators to sample from arbitrary univariate non-uniform
+continuous and discrete distributions have been added to the new
+scipy.stats.sampling submodule. Implementations of a C library
```

Comment:
single backticks around the module name?

Reply:
added, thanks